src/Storages/MergeTree/ReplicatedMergeTreeRestartingThread.cpp

```diff
@@ -30,15 +31,10 @@ namespace ErrorCodes
 
 }
 
-namespace
-{
-    constexpr auto retry_period_ms = 1000;
-}
-
 /// Used to check whether it's us who set node `is_active`, or not.
 static String generateActiveNodeIdentifier()
 {
-    return "pid: " + toString(getpid()) + ", random: " + toString(randomSeed());
+    return Field(ServerUUID::get()).dump();
```

Comment:
I decided to change this to use the server UUID only so it's faster to recover also after unclean shutdowns. There might be better approaches